app/lib/otis/log_importer.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require "date"
+require "json"
+require "zinzout"
+
+module Otis
+  class LogImporter
+    attr_reader :user_map
+
+    def self.logs
+      glob = File.join(
+        ENV.fetch("ULIB_LOGS_ARCHIVE", "/ulib-logs/archive"),
+        "{macc,ictc}*/var/log/babel/access-imgsrv_downloads.log*"
```

Comment:
We'll have to rethink this part since we won't be able to mount it directly

Reply:
Then we could do rclone copy or maybe just rclone cat for each relevant log file.